nd4j/nd4j-backends/nd4j-backend-impls/nd4j-native/src/main/java/org/nd4j/nativeblas/Nd4jCpu.java

```diff
@@ -12994,6 +12994,7 @@ public native void scatterUpdate(@Cast("Nd4jPointer*") PointerPointer extraPoint
 // #include <ops/declarable/headers/BarnesHutTsne.h>
 // #include <ops/declarable/headers/images.h>
 // #include <ops/declarable/headers/updaters.h>
+// #include <ops/declarable/headers/decoder.h>
```

Comment:
Ideally changes to Nd4Cpu.java should be included in the original Pull Request that adds the C++ ops.

Reply:
@treo Nd4jCpu.java is actually generated from nd4j-cpu-presets. This should actually just be on .gitignore.